src/DevBetterWeb.Core/Interfaces/IMarkdown.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DevBetterWeb.Core.Interfaces
+{
+  public interface IMarkdown
```

Comment:
Interface name should describe what it does/what it is. If nothing else `IMarkdownService` at least tells me I should expect it to be a service I DI into things. `IMarkdownConverter` or `IMarkdownRenderer` might be good options as well.

Reply:
renamed to IMarkdownService